test/ServerTest.php

```diff
@@ -129,7 +136,7 @@ public function testListenEmitsStatusHeaderWithoutReasonPhraseIfNoReasonPhrase()
             'QUERY_STRING' => 'bar=baz',
         ];
 
-        $callback = function ($req, $res) {
+        $callback = function (ServerRequestInterface $req, ResponseInterface $res) {
```

Comment:
I'm assuming it's for IDE support, which is likely the only reason to add the property declarations as well.